src/log.c

```diff
@@ -0,0 +1,96 @@
+/* copyright 2012 - 2013 Sascha Kruse and contributors (see LICENSE for licensing information) */
+
+#include "log.h"
+
+#include <glib.h>
+
+static GLogLevelFlags log_level = G_LOG_LEVEL_WARNING;
+
+static const char *log_level_to_string (GLogLevelFlags level)
```

Comment:
There shouldn't be a space after the function name, we only use space before parenthesis after keywords.

Similarly with the other functions in this file.